incubator/docker-registry/values.yaml

```diff
@@ -24,3 +25,26 @@ persistentVolume:
   ##
   storageClass: "-"
   annotations: {}
+
+fqdn: registry.docker.example.com
```

Comment:
Can you change this to just be `tlsSecretName` or something. It is only used for that at the moment and this may be confused with a setting for ingress.